x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomActionsOverview/CustomActionsFlyout/ActionSection.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import {
+  ValidationOptions,
+  FormContextValues,
+  NestDataObject
+} from 'react-hook-form';
+import {
+  EuiTitle,
+  EuiSpacer,
+  EuiText,
+  EuiFieldText,
+  EuiFormRow
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { isEmpty } from 'lodash';
+import { CustomActionFormData } from './';
+
+interface ActionField {
+  name: keyof CustomActionFormData;
+  label: string;
+  helpText: string;
+  placeholder: string;
+  register: ValidationOptions;
+}
+
+const actionFields: ActionField[] = [
+  {
+    name: 'label',
+    label: 'Label',
+    helpText:
+      'Keep it as short as possible. This is the label shown in the actions context menu.',
```

Comment:
Maybe flip this sentence:
```suggestion
      'This is the label shown in the actions context menu. Keep it as short as possible.',
```